### PR TITLE
Rollback to specific release version

### LINKED ISSUE #10: How to rollback to a particular release?
Using Ansistrano rollback, we can rollback to the last release and with the subsequent rollback we can rollback to the n-1 release. But I want know how to rollback to any specific release directly?

### PR DESCRIPTION
**Pull request by tonyo**
Hi guys, thanks for Ansistrano!

This PR adds a feature to rollback to some specific release version (therefore, it fixes #10). I would like to hear someone's opinion/feedback on the approach first, so I haven't updated READMEs yet. 
Essentially (and unsurprisingly), the idea is to specify the release you want to rollback to (e.g., "20170305132734Z") in the `ansistrano_rollback_to_release` variable. If the version exists, we set the symlink to it.

### CODE REVIEW

File: tasks/setup.yml [hunk 1]
```diff
@@ -16,10 +16,25 @@
...
+  when: ansistrano_rollback_to_release != "" and (stat_rollback_release_version.stat.exists is not defined or stat_rollback_release_version.stat.isdir == False)
+
 - name: ANSISTRANO | Get previous releases version
   shell: echo `ls -1t {{ ansistrano_releases_path.stdout }} | head -n 2 | tail -n 1`
   register: ansistrano_previous_release_version
+  when: ansistrano_rollback_to_release == ""
+
+- name: ANSISTRANO | Get rollback release version
+  set_fact:
+    ansistrano_rollback_release_version: "{{ ansistrano_rollback_to_release or ansistrano_previous_release_version.stdout }}"
```
Comment (ricardclau) [inline tasks/setup.yml]: Not sure if this or expression always works, I would prefer an if else statement similar to https://github.com/ansistrano/deploy/blob/master/tasks/update-code/git.yml#L33

**Comment by ricardclau** [comment]
I like the approach although we need to tag a 2.x series as ansistrano_previous_release_version is not always defined now

File: tasks/setup.yml (hunk 1 above)
Comment (tonyo) [reply tasks/setup.yml]: Updated.

**Comment by tonyo** [comment]
@ricardclau We can compute `ansistrano_previous_release_version` unconditionally (without looking at `ansistrano_rollback_to_release`), so it will always be defined. What do you think?

**Comment by ricardclau** [comment]
I got confused with a previous PR, the variable that is often needed is `ansistrano_release_path` as this can be used to run post-update hooks and so on

However, I agree with you, having `ansistrano_previous_release_version` regardless of this new method can be handy as well. Could you please add it?

Also... in order to avoid problems in testing, can you also modify travis.yml removing version 1.8 and putting the same ansible versions you can see in https://github.com/ansistrano/deploy/blob/master/.travis.yml#L6:L8 and also modify https://github.com/ansistrano/rollback/blob/master/meta/main.yml#L7 to reflect Ansible 1.9 as minimum supported?

Thanks for your work on this issue. Looking great so far!

**Comment by tonyo** [comment]
@ricardclau Updated. 
> Thanks for your work on this issue.

My pleasure!

**Review by ricardclau — APPROVED** [review]

**Pull request merged**